pennylane_lightning/lightning_qubit/_state_vector.py

```diff
@@ -49,43 +52,15 @@ class LightningStateVector:
     """
 
     def __init__(self, num_wires, dtype=np.complex128, device_name="lightning.qubit"):
```

Comment:
We don't need the device name as an input variable anymore. The original idea was to have the device_name as the agent responsible for the multi-dispatching. As we move away from this approach, the device name can be an internal attribute, if needed.

Reply:
I will remove and change this.